src/features/geography/l10n/messageIds.ts

```diff
@@ -0,0 +1,34 @@
+import { m, makeMessages } from 'core/i18n';
+
+export default makeMessages('feat.geography', {
```

Comment:
Seeing this makes me second guess what I said about a "geography" feature. Because the map is indeed part of a geography "feature" rather than an "areas" feature.

I'm beginning to think that we should have all of these features:

* Areas (the data, drawing, etc)
* Area assignments (everything under the assignment, including it's map etc)
* Canvass (the frontend)
* Geography (the section and general-purpose map)